app/models/settings.rb

```diff
@@ -106,7 +110,12 @@ def default_account_plan
   end
 
   def update_approval_required(attrs)
-    value = attrs.delete(:account_approval_required) || false
+    value = attrs.delete(:account_approval_required).presence || false
```

Comment:
This prevents `Mysql2::Error: Column 'approval_required' cannot be null`, on the Plan model when the value is empty.

Reply:
if this is a boolean, why not use `present?`?

Reply:
> if this is a boolean, why not use `present?`?

It can (and most likely will often) be a string `"false"`, and we need to keep it this way.